tests/by-util/test_chgrp.rs

```diff
@@ -83,9 +83,10 @@ fn test_preserve_root() {
         new_ucmd!()
             .arg("--preserve-root")
             .arg("-R")
-            .arg("bin").arg(d)
+            .arg("bin")
+            .arg(d)
             .fails()
-            .stderr_is("chgrp: it is dangerous to operate recursively on '/'\nchgrp: use --no-preserve-root to override this failsafe");
+            .stderr_contains("chgrp: it is dangerous to operate recursively");
```

Comment:
why did you remove the full error message? thanks

Reply:
ping?

Reply:
Sorry for the delay in responding! In GNU, `chgrp --preserve-root -R bin /` prints the error "chgrp: it is dangerous to operate recursively on '/'"

If the path is indirect, GNU adds more context. `chgrp --preserve-root -R bin /tmp/..` prints "chgrp: it is dangerous to operate recursively on '/tmp/..' (same as '/')"

I added the parenthetical for indirect paths that resolve to `/`, for consistency with GNU. Since different test paths now give different error messages, checking for an exact match against a static string no longer works.